.appveyor.yml

```diff
@@ -12,12 +12,12 @@ image:
   - Visual Studio 2017
 
 environment:
-  nodejs_version: "4"
+  nodejs_version: "6"
   matrix:
     - PLATFORM: windows-10-store
       JUST_BUILD: --justBuild
 install:
-  - npm cache clean -f
+  - ps: Install-Product node $env:nodejs_version
```

Comment:
What is this change? Better way to select the node version?

Reply:
Ouch, so our previous testing setup on AppVeyor was actually broken!? D'oh.

Reply:
More or less ;)